.CMake/alg_support.cmake

```diff
@@ -1,7 +1,11 @@
 include(CMakeDependentOption)
 
+if(NOT OQS_KEM_DEFAULT)
 set(OQS_KEM_DEFAULT "OQS_KEM_alg_frodokem_640_aes")
+endif()
+if(NOT OQS_SIG_DEFAULT)
```

Comment:
Same here, with `NOT DEFINED OQS_SIG_DEFAULT`

Reply:
Done